cloudify_cli/commands/executions.py

```diff
@@ -29,6 +29,11 @@
 from cloudify_cli.logger import get_events_logger
 
 
+_STATUS_CANCELING_MESSAGE = (
+    'NOTE: Executions currently in a "canceling status" '
+    'may take a while to complete')
+
```

Comment:
NOTE: Executions currently in a "canceling/force-canceling" status may take a while to change into "cancelled"